### ISSUE TITLE
add control over finetuning

### ISSUE THREAD

**Issue by bw4sz** [weecology/DeepForest#15]
Depth of layers in DeepForest/retinanet_train.py

Need to be able to specify which layers to frees before compiling.

reduce the learning rate.

https://flyyufelix.github.io/2016/10/03/fine-tuning-in-keras-part1.html

**Comment by bw4sz** [comment]
Just merge the boxes in the overlapping area, not use IoU, it will penalize one of the boxes too much.
